etc/kayobe/kolla/passwords.yml

```diff
@@ -1,1010 +1,1010 @@
 $ANSIBLE_VAULT;1.1;AES256
```

Comment:
Do you know what has changed in this file? You can use ansible-vault decrypt and diff to check.